service/static/js/rest_api.js

```diff
@@ -11,18 +11,41 @@ $(function () {
         $("#customer_last_name").val(res.last_name);
         $("#customer_email").val(res.email);
         $("#customer_password").val(res.password);
+        $("#customer_address_id").val(res.addresses[0].address_id);
         $("#customer_street").val(res.addresses[0].street);
         $("#customer_city").val(res.addresses[0].city);
         $("#customer_state").val(res.addresses[0].state);
         $("#customer_country").val(res.addresses[0].country);
         $("#customer_pin_code").val(res.addresses[0].pin_code);
+       
         if (res.active == true) {
             $("#customer_active").val("true");
         } else {
             $("#customer_active").val("false");
         }
     }
 
+    // Updates the form with data from two responses
+    function update_form_data_two_responses(res1,res2) {
+        $("#customer_id").val(res1.id);
+        $("#customer_first_name").val(res1.first_name);
+        $("#customer_last_name").val(res1.last_name);
+        $("#customer_email").val(res1.email);
+        $("#customer_password").val(res1.password);
+        $("#customer_address_id").val(res2.address_id);
```

Comment:
displayFieldRequiredNotification if address_id is missing.

Reply:
added